spec/system/sdg/goals_spec.rb

```diff
@@ -140,4 +140,25 @@
       end
     end
   end
+
+  describe "Help" do
+    scenario "shows all SDGs targets" do
+      create(:sdg_local_target, code: "15.1.1", title: "SDG local target sample text")
+      visit sdg_help_path
+
+      click_link "7. Affordable and Clean Energy"
```

Comment:
How about an expectation about the initial text, before users click any tab? :thinking:

Reply:
Unresolving; what I meant was an expectation about the initial `h2` :wink:.

By the way, the test would pass if we displayed the information for all 17 goals all the time, right? IMHO we should also test that after clicking a link, the previous `h2` is not available anymore :thinking:.